Factory/PsrHttpFactory.php

```diff
@@ -67,12 +68,22 @@ public function createRequest(Request $symfonyRequest)
 
         $body = $this->streamFactory->createStreamFromResource($symfonyRequest->getContent(true));
 
+        if (method_exists(Request::class, 'getPayload')) {
+            try {
+                $parsedBody = $symfonyRequest->getPayload()->all();
```

Comment:
The PSR-7 standard is very explicit about when not to parse the body:

> If the request Content-Type is either `application/x-www-form-urlencoded` or `multipart/form-data`, and the request method is `POST`, this method MUST return the contents of `$_POST`.

https://github.com/php-fig/http-message/blob/402d35bcb92c70c026d1a6a9883f06b2ead23d71/src/ServerRequestInterface.php#L155-L157

Reply:
I've fixed all your feedback